mpi-proxy-split/p2p_drain_send_recv.cpp

```diff
@@ -84,7 +85,12 @@ registerLocalSendsAndRecvs()
   g_bytesSentToUsByRank[g_world_rank] = 0;
 
   // Free resources
+  // Semantics -- although mana_comm IS a real id, and MPI_Comm_free_internal does look up a virtual id to remove, since it cannot find the virtual id, the real id is returned back and freed in the lh. So there is no leak with mana_comm.
```

Comment:
Pls reformat for 80 character lines.
Minor changes:
`to remove, since it` -> `to remove. Since it`
`no leak with mana_comm` -> `no memory leak caused by mana_comm`